src/routes/index.svelte

```diff
@@ -1,6 +1,70 @@
 <script>
+    import { onMount } from 'svelte'
     import { fetchFromApi } from '../utils/fetchFromApi'
 
+    let canvas
+    let chartWidth = 800
+    let chartHeight = 500
+    let xOffset = 10
+    let yOffset = 10
+    let radius = 3
+    let startAngle = 0
+    let endAngle = 2 * Math.PI
+
+    function drawCanvas() {
+        const ctx = canvas.getContext('2d')
+
+        ctx.beginPath()
+        ctx.lineWidth = '3'
+        ctx.strokeStyle = 'black'
+        ctx.fillStyle = 'white'
+        ctx.rect(0, 0, chartWidth, chartHeight)
+        ctx.fillRect(0, 0, chartWidth, chartHeight)
+        ctx.stroke()
+    }
+
+    function drawGraph(history) {
+        const ctx = canvas.getContext('2d')
+        ctx.clearRect(0, 0, chartWidth, chartHeight)
+        drawCanvas()
+
+        const allValues = history
+            .flat()
+            .map(data => data.close)
+            .flat()
+        let minYValue = Math.min(...allValues)
+        let maxYValue = Math.max(...allValues)
+
+        const colours = ['red', 'blue', 'green']
```

Comment:
Once we know how many stocks we are going to show max, we should add that many colors into this object.